std/experimental/allocator/building_blocks/bitmapped_block.d

```diff
@@ -246,6 +251,15 @@ struct BitmappedBlock(size_t theBlockSize, uint theAlignment = platformAlignment
         }
     }
 
+    /*
+    Based on the latest allocated bit, 'newBit', it adjusts '_freshBit'
+    */
+    void adjustFreshBit(const ulong newBit)
+    {
+        import std.algorithm.comparison : max;
+        _freshBit = max(newBit, _freshBit);
```

Comment:
`if (newBit > _freshBit) _freshBit = newBit;` seems to do better for performance, see https://goo.gl/WeQ1KS and https://godbolt.org/g/YUnzNw